backend/src/app.controller.ts

```diff
@@ -107,18 +76,38 @@ export class AppController {
   }
 
   /**
-   * Public endpoint to get list of ministries
-   * Used by CHEFS form
-   * @returns {Ministry[]} List of ministries
+   * Return lists of values used by frontend and CHEFS form
+   * @param req
+   * @returns
    */
   @Public()
-  @Get('/ministries')
-  getMinistries() {
+  @Get('program-field-data')
```

Comment:
We had talked about identifying in the request what fields we want to request. Still want to do this?